docs/unattend.xml

```diff
@@ -2,7 +2,7 @@
 <!--A sample unattend.xml which helps in setting admin password and running scripts on first boot-->
 <unattend xmlns="urn:schemas-microsoft-com:unattend">
    <settings pass="specialize">
-      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http:// www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-International-Core" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
+      <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-International-Core" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
```

Comment:
I suggest collapsing this commit into the other commit where you are changing this line and calling it out in the commit message.

Reply:
Mission accomplished!